foundations/likvid-prod/meshstack/terragrunt.hcl

```diff
@@ -0,0 +1,72 @@
+terraform {
+  source = "${get_repo_root()}//kit/foundation/meshstack"
+
+  # todo: we should source that from a foundation.yml somehow?
+  extra_arguments "profile" {
```

Comment:
d: add comment 

need to ensure authentication against aws so we can fetch remote_state from S3 for the aws plattform